src/components/layouts/landingPage/ArticuleSection/index.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import HeadingText from 'Atoms/HeadingText';
+import Image from 'Atoms/Image';
+import __fontSizes from 'FontSizes';
+import __devices from 'Devices';
+
+const ArticulesSection = () => {
+  const ROBOT_IMAGE =
+    'https://res.cloudinary.com/maintenance-site/image/upload/v1580117293/Ojumo/2EJCSULRwC8.png';
+  const CHILDREN_IMAGE =
+    'https://res.cloudinary.com/maintenance-site/image/upload/v1580117292/Ojumo/AEaTUnvneik.png';
+  const articules = [
+    {
+      imageURL: ROBOT_IMAGE,
+      title: 'Effects of AI and ML',
+      body:
+        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, modi quod dolorem cumque iure totam blanditiis accusantium amet quia facilis ipsum distinctio tempora? Soluta.',
+      date: 'Date - 20th May 2019',
+      time: 'Time - 8am - 4pm'
+    },
+    {
+      imageURL: CHILDREN_IMAGE,
+      title: 'Extracts from our visit to the ophanage',
+      body:
+        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, modi quod dolorem cumque iure totam blanditiis accusantium amet quia facilis ipsum distinctio tempora? Soluta.',
+      date: 'Date - 20th May 2019',
+      time: 'Time - 8am - 4pm'
+    }
+  ];
```

Comment:
I think this array should have the keys:
- imageURL
- title
- author
- datePublished
- description

review the mocks to see how these fields are used

Reply:
can't find where this is specified in the mocks
I have updated but can't find where author is used